python/lsst/ctrl/execute/libexec/dagIdInfo.py

```diff
@@ -56,3 +57,7 @@
         sys.exit(0)
     file.close()
```

Comment:
I don't know how I missed this in the past but with should, of course, be done with `with open()` these days.